R/map-if-at.R

```diff
@@ -20,8 +20,9 @@
 #'   `TRUE` will be modified.
 #' @param .else A function applied to elements of `.x` for which `.p`
 #' returns `FALSE`.
-#' @export
+#' @inheritSection map Parallelization
```

Comment:
Maybe this could now just be a link in the documentation for `.f`?

Reply:
Agree that this is clearer, and amended in [c47a161](https://github.com/tidyverse/purrr/pull/1185/commits/c47a16146f5ba981aa8b5c3900e3d2364206c9fa).